### PR TITLE
build navbar component

### PR DESCRIPTION
**Pull request by Rockyraven**

### CODE REVIEW

File: component/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    width: 15rem;
+    margin-bottom: 2rem;
+}
+.search-input{
+    height: 2rem;
+    width: 17rem;
+}
+.search-button{
+    position: absolute;
+    margin: 4px 15rem
```
Comment (NeerajVodala) [inline component/navbar/navbar.css]: use any one unit here

File: component/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    justify-content: center;
+    align-items: center;
+    border: 1px solid;
+    padding: 5px;
+}
+.login-section a,.search-bar a{
+   color: var(--secondary-color);
+   text-decoration: none;
+   font-weight: bold;
+   margin-right: 10px;
```
Comment (NeerajVodala) [inline component/navbar/navbar.css]: Try using only unit(preferably rem)

**Review by NeerajVodala — COMMENTED** [review]
Be consistent with the units.
Rest everything looks fine to me.
All the best.

**Review by shashikantmaurya0007 — APPROVED** [review]
Be consistent with  the unit .rest looks good to merge;)

File: component/navbar/navbar.css (hunk 1 above)
Comment (Rockyraven) [reply component/navbar/navbar.css]: I have changed unit

**Pull request merged**